### DISCUSSION TITLE
Настройка списка проектов.

### DISCUSSION THREAD

**Discussion by SerafimD** [uPagge/gitlab-notification#12] [Ideas]
Предлагаю добавить в бота функционал настройки списка проектов : 
1. При старте приложения не по 2 - ум категориям private/public а с возможностью более тонкой настройки.
2. Уже в процессе работы могут появляться новые репы, которые было - бы неплохо добавить в модель project.

Пока можно только отредактировать БД как я понимаю.

**Comment by uPagge** [comment]
Привет
Пока нет времени заниматься проектом, но многое хочется там переделать.
Надеюсь летом появится время на это, обязательно рассмотрю это предложение.
